protocols/discovery/src/lib.rs

```diff
@@ -163,7 +163,7 @@ pub struct Discovery<M> {
 
     dynamic_query_cycle: Option<Duration>,
 
-    check_interval: Interval,
+    check_interval: Option<Interval>,
```

Comment:
Why this is changed to an option?

Reply:
tokio's time now depends on the runtime global handle and cannot be created outside the runtime